content/en/getting_started/agent/_index.md

```diff
@@ -13,32 +13,61 @@ further_reading:
       text: 'Why should I install the Datadog Agent on my cloud instances?'
 ---
 
-This guide provides an introduction to the Agent and how you can use it to send system-level metrics to the Datadog platform. It walks through an example Agent installation on Ubuntu. It covers:
+## Overview
 
-  - Agent installation
-  - Verifying that the Agent is running
-  - Configuring Agent features
-  - Troubleshooting resources
+This guide introduces the Datadog Agent and covers:
 
-## Overview
+  - [Introduction to the Agent](#the-datadog-agent)
+  - [Installing the Agent](#installing-the-agent)
+  - [Data collected by the Agent](#data-collected-by-the-agent)
+  - [Advanced configurations and features](#advanced-configurations-and-features)
+  - [Troubleshooting the Agent](#troubleshooting)
+
+
+## What is the Datadog Agent
+
+The Datadog Agent is a software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. 
 
-### About the Agent
+The Agent can run on:
+- Your local hosts (Windows, MacOS), 
+- A containerized environments (Docker, Kubernetes),
+- In on-premises data centers. 
 
-The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. It can run on your local hosts (Windows, MacOS), containerized environments (Docker, Kubernetes), and in on-premises data centers. You can install and configure it using configuration management tools (Chef, Puppet, Ansible).
+You can also install and configure the Agent using configuration management tools like Chef, Puppet, or Ansible.
 
 The Agent is able to collect 75 to 100 system-level metrics every 15 to 20 seconds. With additional configuration, the Agent can send live data, logs, and traces from running processes to the Datadog Platform. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][1].
```

Comment:
I know you didn't work on this part, but there's a few things worth changing here for editorial style:
```suggestion
The Agent can collect 75-100 system-level metrics every 15-20 seconds. With additional configuration, it can send live data, logs, and traces from running processes to Datadog. The Datadog Agent is open source, and its source code is available on GitHub at [DataDog/datadog-agent][1].
```